packages/builder/src/stores/portal/auth.js

```diff
@@ -172,6 +172,14 @@ export function createAuthStore() {
         resetCode,
       })
     },
+    generateAPIKey: async () => {
+      const info = await API.generateAPIKey()
+      return info?.apiKey ? info.apiKey : null
+    },
+    fetchAPIKey: async () => {
+      const info = await API.fetchDeveloperInfo()
+      return info?.apiKey ? info.apiKey : null
```

Comment:
Same as above above oprtional chaining, but this one looks like it will return more info than just an API key so that part doesn't apply here.